<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindInFilesDialog</class>
 <widget class="QDialog" name="FindInFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find in Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="findLabel">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="findLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fileTypesLabel">
       <property name="text">
        <string>File types</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="typesLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="directoryLabel">
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="caseSensitiveLabel">
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="casedCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="recursiveLabel">
       <property name="text">
        <string>Recursive</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="recursiveCheckBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="dirLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="dirToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Find results</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListWidget" name="resultsListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findButton">
       <property name="text">
        <string>Find All</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>findButton</sender>
   <signal>clicked()</signal>
   <receiver>FindInFilesDialog</receiver>
   <slot>find()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirToolButton</sender>
   <signal>clicked()</signal>
   <receiver>FindInFilesDialog</receiver>
   <slot>browseForDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resultsListWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>FindInFilesDialog</receiver>
   <slot>showResult(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FindInFilesDialog</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>find()</slot>
  <slot>browseForDir()</slot>
  <slot>showResult(QListWidgetItem*)</slot>
  <slot>cancel()</slot>
 </slots>
</ui>
